common/vendor/MarkdownEditor/angular-markdown-editor.js

```diff
@@ -0,0 +1,200 @@
+angular

```

Comment:
It is a third party code that I have modified to suit our requirements. I will write header description for the directive.